python/ray/data/datasource/tfrecords_datasource.py

```diff
@@ -23,13 +33,23 @@ def __init__(
         self,
         paths: Union[str, List[str]],
         tf_schema: Optional["schema_pb2.Schema"] = None,
+        fast_read: bool = False,
```

Comment:
Can we change the name `fast_read` to `read_with_tfx`? 
The `fast_read` looks a bit vague to me.